python/SoapySDR.in.i

```diff
@@ -72,11 +72,53 @@ if os.name == 'nt' and hasattr(os, 'add_dll_directory'):
 ////////////////////////////////////////////////////////////////////////
 %include <SoapySDR/Config.h>
 
+////////////////////////////////////////////////////////////////////////
+// Check ABI before attempting to use Python module
+////////////////////////////////////////////////////////////////////////
+%include <std_string.i>
+
+%{
+#include <SoapySDR/Version.hpp>
+
+std::string getPythonModuleABIVersion()
+{
+    return "@SOAPY_SDR_ABI_VERSION@";
+}
```

Comment:
Take a look at `python/CMakeFiles/_SoapySDR.dir/SoapySDRPYTHON_wrap.cxx`

`SWIG_Python_SetConstant(d, "SOAPY_SDR_ABI_VERSION",SWIG_FromCharPtr("0.8"));`

What I was saying is swig is actually already building this exact string into the C++ source and making it available in python. It was always there ripe for the taking. You dont have to do anything. Other than including Version.h earlier for the check to be earlier I guess.

Reply:
No more fanciness, got it.

Reply:
The checks where python only, so I didnt want the other build changes. Forgot I can push to branch, its on this repo afterall. Anything sus before I merge this and tag it? It looks like the gnuradio folks are going to build this software all of the time now, im not sure but I think I can just make out the avalanche in the distance.

Reply:
This should do it.